styles/variables-silicon.less

```diff
@@ -141,7 +141,11 @@
 
 // Heading
 // ---------------------------------------
-@agate-heading-padding: 3px 0;
+@agate-heading-height: 60px;
+@agate-heading-margin: 0;
+@agate-heading-padding: 0 30px 0 9px;
+@agate-heading-line-height: @agate-heading-height;
+@agate-heading-width: 660px;
```

Comment:
I don't think this is necessary. It looks natural when the heading has no width and the container has width.
Please remove it.